### DISCUSSION TITLE
please add COMMENT of tag editable function in webpage

### DISCUSSION THREAD

**Discussion by sensen1** [navidrome/navidrome#2456] [Ideas]
For file types like mp3, add editing capabilities to the commet attribute in webpage. This would allow indirectly realizing lyrics functionality  or other  information show.

**Comment by metalheim** [comment]
Navidrome takes the attribute from your files tags. You can edit the "comment" tag of your files and navidrome will reflect that. 
If all files of an album have the same comment, navidrome will interpret it as an album comment.
Same with lyrics. If you tag the lyrics in the proper lyrics tag, navidrome will pick these up.

I see that editing a comment/commenting with the WebUI would be user-friendly , but that means that the navidrome database and the tags of your files would be out-of-sync. 
- What should happen if you commented through WebUI and then changed the comment tag in a file?
